examples/command/command/app.py

```diff
@@ -135,7 +136,7 @@ def action4(widget):
 
         # The order in which commands are added to the app or the toolbar won't
         # alter anything. Ordering is defined by the command definitions.
-        self.app.commands.add(cmd1, cmd0, cmd6, cmd4, cmd3)
+        self.app.commands.add(cmd1, cmd0, cmd6, cmd4, cmd3, cmd7)
```

Comment:
*Not* adding cmd7 here was an explicit test of the "adding to toolbar implies adding to app" case.

Reply:
Reverted.

Reply:
On second thoughts, I think it's better to add one of the commands to both places to verify that doesn't cause any problems. And I've added a similar case to the core tests.

Reply:
As long as we've got an example of all three possibilities (app only, toolbar only, app and toolbar), 👍.